l/libbytesize/stone.yaml

```diff
@@ -0,0 +1,33 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : libbytesize
+version     : 2.10
```

Comment:
version literal needs to be quoted; otherwise YAML will interpret it as `2.1` (see associated jsonc manifest `source-release` value above).